.gitignore

```diff
@@ -328,3 +328,4 @@ ASALocalRun/
 
 # MFractors (Xamarin productivity tool) working folder 
 .mfractor/
```

Comment:
Sorry to be pedantic, but could the ionide ignore have a comment before it like the others please?

Reply:
Sure